javascript/javascript1/week2/weather-wear.js

```diff
@@ -0,0 +1,9 @@
+function weatherWear(temperature) {
+if (temperature <= 15 ) {
+  return 'stay worm and wear a jacket';
+} 
+else if  (temperature > 15 && temperature <= 30) {
+  return 'injoy weather and wear shorts a t-shirt';
+}
+}
+console.log(weatherWear(15));
```

Comment:
Commit message you give should be related to the file you have committed.

ex: Here I see the weather-wear.js file and the function related to it, but the commit message is about "flight booking"